docs/cambridge.md

```diff
@@ -5,10 +5,58 @@ To use, run the pipeline with `-profile cambridge`. This will download and launc
 with a setup suitable for the Cambridge HPC cluster. Using this profile, either a docker image containing all of the required software will be downloaded,
 and converted to a Singularity image or a Singularity image downloaded directly before execution of the pipeline.
 
-The latest version of Nextflow is not installed by default on the Cambridge HPC cluster. You will need to install it into a directory you have write access to.
-Follow these instructions from the Nextflow documentation.
+### Install Nextflow
 
-- Install Nextflow : [here](https://www.nextflow.io/docs/latest/getstarted.html#)
+The latest version of Nextflow is not installed by default on the Cambridge HPC cluster CSD3. You will need to install it into a directory you have write access to.
+Follow [these instructions](https://www.nextflow.io/docs/latest/getstarted.html#) from the Nextflow documentation.
+
+```
+# move to desired directory on HPC
+cd /home/<username>/path/to/dir
+# get the newest version
```

Comment:
Could provide alternative instructions e.g. with conda?

Reply:
Good idea. I have added a section for conda here [1f73a79](https://github.com/nf-core/configs/commit/1f73a79c78029c656c5bdedb5ee066a83e31f766).